test/map_notifier_test.dart

```diff
@@ -39,38 +48,53 @@ void main() {
     test('Listener is notified on clear', () {
       mapNotifier.addAll(newValues);
       expect(result, newValues);
+
+      /// Delete all the keys and values in mapNotifier
       mapNotifier.clear();
+
+      /// Check if mapNotifier is empty
       expect(result.isEmpty, isTrue);
     });
 
     test('Listener is notified on putIfAbsent', () {
-      for (final entry in newValues.entries) {
```

Comment:
never do this. Always use `{}` even even for a single line. Because the moment you add a second line it is so easy to forget to add the `{` and then you are surprised why it doesn't work